Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -1493,12 +1493,26 @@ public struct ClusterInvocationResultHandler: DistributedTargetInvocationResultH
             system.log.debug("Result handler, onThrow: \(error)", metadata: ["call/id": "\(callID)"])
             let reply: RemoteCallReply<_Done>
             if let codableError = error as? (Error & Codable) {
-                reply = .init(callID: callID, error: codableError)
+                switch system.settings.remoteCall.codableErrorAllowance {
+                case .custom(let allowedTypes) where errorTypeAllowed(error: error, allowedTypes: allowedTypes):
+                    reply = .init(callID: callID, error: codableError)
+                case .all: // compiler gets confused if this is grouped together with above
+                    reply = .init(callID: callID, error: codableError)
+                default:
+                    reply = .init(callID: callID, error: GenericRemoteCallError(message: "Remote call error of [\(type(of: error as Any))] type occurred"))
```

Comment:
does this emit a warning? May have to be `any Any`? (not sure, just asking)

Reply:
I haven't noticed any. Will double check.

Reply:
Don't see any. 👌

Reply:
Ok cool! Learning the new rules about spelling types myself still huh